docker-compose.yml

```diff
@@ -3,7 +3,7 @@ version: '3'
 
 services:
     bot:
-        image: node:13
+        image: node:16
```

Comment:
No idea if this is right

Reply:
I think, this should be because discord.js has minimum requirement for node 16.

Reply:
Yup, I have to do a similar thing [here](https://github.com/ckiee/daiko/blob/171570b51539b68f0716dd5f9ff6c922ceace0d4/default.nix#L5-L8).